test/e2e/multikueue/suite_test.go

```diff
@@ -62,4 +63,10 @@ var _ = ginkgo.BeforeSuite(func() {
 	k8sWorker2Client = util.CreateClientUsingCluster("kind-" + worker2ClusterName)
 
 	ctx = context.Background()
+
+	waitForAvailableStart := time.Now()
+	util.WaitForKueueAvailability(ctx, k8sManagerClient)
+	util.WaitForKueueAvailability(ctx, k8sWorker1Client)
+	util.WaitForKueueAvailability(ctx, k8sWorker2Client)
+	ginkgo.GinkgoLogr.Info("Kueue is Available in all the clusters", "waitingTime", time.Since(waitForAvailableStart))
```

Comment:
This is mostly relevant when we need to recompute the timeout (hopefully never).

Reply:
I agree with @trasc

Reply:
Sure, it is a nit but I think the time that is measured might be confusing to interpret when looking in the logs. It isn't really the timeout, because timeout is per cluster, but the log is for all clusters. 

Anyway, it is a nit, so I'm ok to keep as is if you agree.